code/game/objects/items/mountable_frames/frames.dm

```diff
@@ -25,3 +25,16 @@
 			user << "<span class='rose'>[src] cannot be placed in this area.</span>"
 			return
 		return 1
+
+/obj/item/mounted/frame/newscaster
```

Comment:
I guess it's a matter of opinion, I find vending.dm very convenient to read because I can easily jump between lines in the same file. I don't like having tons of tabs open.